qualang_tools/plot/README.md

```diff
@@ -21,3 +21,72 @@ while job.result_handles.is_processing():
     ...
     # The execution will be cleanly interrupted by closing the figure
 ```
+
+## Fitting 
+This tool makes the fitting process much easier! Once you have the data you can:
+
+* Fit the data to a suitable fitting function.
+* Plot the data as well as the fitting function.
+* Extract physical parameters from the fitting.
+* Save the fitting parameters to a json file.
+* Open the json file and save its data to a python dictionary.
+

```

Comment:
I would write somewhere in the readme the list of the available fitting function so that the user knows if the one he/she's looking for is implemented

Reply:
Good idea, I'll add it :)